test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -507,18 +512,58 @@ void RandomCodeOptions::setWeight(eth::Instruction _opCode, int _weight)
 	setWeights();
 }
 
-void RandomCodeOptions::addAddress(Address const& _address)
+void RandomCodeOptions::addAddress(Address const& _address, AddressType _type)
 {
-	addressList.push_back(_address);
+	switch(_type)
+	{
+		case AddressType::ACCOUNT:
+			accountAddressList.push_back(_address);
+			callAddressList.push_back(_address);
+		break;
+		case AddressType::CALLONLY:
+			callAddressList.push_back(_address);
+		break;
+		default:
+		break;
+	}
 }
 
-Address RandomCodeOptions::getRandomAddress() const
+Address RandomCodeOptions::getRandomAddress(AddressType _type) const
 {
-	if (addressList.size() > 0)
+	int index;
+	int random;
```

Comment:
I'd do the same with `index`, too, actually (not reusing it in different `case`s)